scripts/jupyter.sh

```diff
@@ -15,6 +15,7 @@ DOCKER_COMMAND="docker"
 PORT="8888"
 SCRIPT_NAME=$( echo $0 | sed 's#.*/##g' )
 GPU_DEVICE="--gpus all"
+ENV=""
```

Comment:
I'm curious what you use this for?

Reply:
I use this if I want to use a notebook that uses ml4h code that I am actively developing. I can set my PYTHONPATH to my ml4h directory so that I can edit the ml4h code and have it accessible to the notebook. I use something similar when I run docker so that I can edit the ml4h code and have it show up in the docker without having to restart it.